staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/testing/webhook_server.go

```diff
@@ -138,6 +138,13 @@ func webhookHandler(w http.ResponseWriter, r *http.Request) {
 				},
 			},
 		})
+	case "/noop":
+		w.Header().Set("Content-Type", "application/json")
+		json.NewEncoder(w).Encode(&v1beta1.AdmissionReview{
```

Comment:
I'm really surprised this response is accepted by the webhook. It doesn't include the kind or apiVersion or response.uid

Reply:
Yeah, this doesn't set a good example. I'll clean this up as a follow up. Also should have a look at our validation.

Reply:
we're too accepting today. will have to tighten it up for admissionreview v1